lib/cpl.rb

```diff
@@ -227,11 +235,11 @@ def self.all_base_commands
     check_unknown_options!(except: @commands_with_extra_options)
     stop_on_unknown_option!
 
-    def self.validate_options!(options, command_options)
+    def self.validate_options!(options)
       options.each do |name, value|
         raise "No value provided for option '#{name}'." if value.to_s.strip.empty?
 
-        params = command_options.find { |option| option[:name].to_s == name }[:params]
+        params = ::Command::Base.all_options.find { |option| option[:name].to_s == name }[:params]
```

Comment:
Since `command_options` now lacks `::Command::Base.common_options`, we need to use `::Command::Base.all_options` to ensure that all options can be found.